README.md

```diff
@@ -1,2 +1,32 @@
-# edgepi-rpc-client-py
-EdgePi RPC Client Python
+![Image](https://user-images.githubusercontent.com/3793563/207438826-bb656ca5-f19d-4699-8cb4-35acccb2ce58.svg)
+
+# EdgePi RPC Client
+
+The RPC client exposes functionality of the EdgePi Python SDK over a network, allowing for remote control of the EdgePi.
+
+Since the client directly mirrors the SDK, you can take advantage of the user-friendly SDK functionality with the flexibility of a network connection.
+
+Learn more about the Python SDK [here](https://github.com/EdgePi-Cloud/edgepi-python-sdk).
+
+# Using the RPC Client
+
+Install the RPC client through the terminal:
+
+```
+$ python3 -m pip install edgepi-rpc-client
+```
+
+Once installed, you can control the modules of the EdgePi directly through the SDK's wide functionality.
+
+For example, from a remote connection with address `localhost` and port `5555`, initialize a module as:
+
+```
+from edgepi_rpc_client.services.adc_service import ClientAdcService
+
+# initialize ADC Client
+adc_client = ClientAdcService('tcp://localhost:5555')
+```
```

Comment:
not a big deal, but you can add `python` after the first "```" to make it color coded and easier to read in the actual README:

```python
 from edgepi_rpc_client.services.adc_service import ClientAdcService

 # initialize ADC Client
 adc_client = ClientAdcService('tcp://localhost:5555')
 ```

Reply:
Made the change, thanks!